src/Microsoft.Health.Dicom.Api/Features/BackgroundServices/DeletedInstanceCleanupWorker.cs

```diff
@@ -57,8 +59,6 @@ public async Task ExecuteAsync(CancellationToken stoppingToken)
                 _deleteMeter.OldestRequestedDeletion.Add(metrics.OldestDeletion.ToUnixTimeSeconds());
                 _deleteMeter.CountDeletionsMaxRetry.Add(metrics.TotalExhaustedRetries);
 
-                await Task.Delay(_pollingInterval, stoppingToken);
```

Comment:
Should we do before the GetMetrics so that if the service is getting started and the background service won't error immediately if the datastore is not available right away?